scania/corporate-ui-site
# code-snippet styling added

**Describe pull-request**  
_Describe what the pull-request is about_
Code snippet Styling added

- [x] - Font
- [x] - Spacing
- [x] - Border


**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: Look at the card in scania/projects

**How to test**  
_Add description how to test if possible_
- Go to getting started > development > scroll to the bottom to see the correct example
- Look at figma

**Screenshots**  
_If applicable, add screenshots to help explain_
![image](https://user-images.githubusercontent.com/35451568/101874246-ce707680-3b88-11eb-9da1-8e576aefc6cf.png)


**Additional context**  
_Add any other context about the pull-request here._
copy function is for later
